src/Tests/OneExplorer/ConfigObject.test.ts

```diff
@@ -46,16 +46,15 @@ suite("OneExplorer", function () {
       });
 
       test("NEG: Create config of a file without any valid content", function () {
-        const configName = "model.cfg";
```

Comment:
I think using const seems to be better for code readability, but it seems to increase CC violations(!!). 
I don't understand why using simple constants/variables is related to cyclomatic complexity score.
😢

Reply:
I totally agree... I just followed the guide anyways. 
In terms of readability, I first thought that using const is better but some people told me that writing down the file name is better... 🤔